### PR TITLE
feat: Refactor the directory sturcture of model bundle

### LINKED ISSUE #7: [discussion] Should We Have the ormbfile.yaml?
Now we have the ormbfile.yaml in the directory of the model. It has some problems:

- Some model servers will return errors when serving the model
- Additional maintenance cost

We should have a final decision before the first release.

/priority p0

### PR DESCRIPTION
**Pull request by gaocegege**
**What this PR does / why we need it**:

- Add an example for PMML
- Refactor the directory structure of the model bundle
  - Mainly move the ormbfile.yaml to the upper directory

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #7 

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @ddysher @codeflitting 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/model/loader.go [hunk 1]
```diff
@@ -40,8 +41,9 @@ func (d defaultLoader) Load(path string) (*Model, error) {
 		return nil, err
 	}
 
+	// Load the model from <path>/model.
 	buf := &bytes.Buffer{}
-	if err := Tar(path, buf); err != nil {
+	if err := Tar(filepath.Join(path, consts.ORMBModelDirectory), buf); err != nil {
```
Comment (codeflitting) [inline pkg/model/loader.go:46]: Should we check the directory format? To ensure that the format in the ormbfile is consistent with what is being passed

**Comment by ddysher** [comment]
/lgtm 

@gaocegege  I'll pair with you before folks are ready to jump into the project.

File: pkg/model/loader.go (hunk 1 above)
Comment (gaocegege) [reply pkg/model/loader.go:46]: I think it is a huge feature. We can open an issue and implement it in the future. WDYT

Comment (ddysher) [reply pkg/model/loader.go:46]: LGTM

issue created https://github.com/caicloud/ormb/issues/11

**Pull request merged**